db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
probably a typo, `drop_table` does not execute blocks.

```ruby
def drop_table(table_name, options = {})
  execute "DROP TABLE #{quote_table_name(table_name)}"
end
```

Source: http://apidock.com/rails/v4.2.1/ActiveRecord/ConnectionAdapters/SchemaStatements/drop_table